### PR TITLE
add example of minicpm

### PR DESCRIPTION
**Pull request by LDLINGLINGLING**
Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [ ] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `conda deactivate; bash -i tests/backward_compatibility_tests.sh`

### CODE REVIEW

File: llm/minicpm/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          "role": "user",
+          "content": "What is the best food?"
+        }
+      ],
+      "max_tokens": 512
+  }' | jq -r '.choices[0].message.content'
+```
+
+
+## **Optional:** Accessing Qwen with Chat GUI
```
Comment (Michaelvll) [inline llm/minicpm/README.md:66]: Should we update the name of the model here?

File: docs/source/_gallery_original/index.rst [hunk 2]
```diff
@@ -35,6 +35,7 @@ Contents
    :caption: LLM Models
 
    Mixtral (Mistral AI) <llms/mixtral>
+   MiniCPM (openbmb) <llms/minicpm>
```
Comment (Michaelvll) [inline docs/source/_gallery_original/index.rst:38]: By adding this here, we need to soft link the readme file at `llms/minicpm/README.md` to `docs/source/_gallery_original/llms/`.

**Review by Michaelvll — COMMENTED** [review]
Thanks for adding the example for MiniCPM @LDLINGLINGLING! We are happy to see a new open model showing up. Could you please include the commands we have tested in the PR description for a record?

File: llm/minicpm/README.md (hunk 1 above)
Comment (LDLINGLINGLING) [reply llm/minicpm/README.md]: Already modified

File: docs/source/_gallery_original/index.rst (hunk 2 above)
Comment (LDLINGLINGLING) [reply docs/source/_gallery_original/index.rst]: Already modified

File: llm/minicpm/README.md (hunk 1 above)
Comment (LDLINGLINGLING) [reply llm/minicpm/README.md]: Already modified

**Pull request closed without merging**